indy_node/test/auth_rule/test_get_auth_rule.py

```diff
@@ -154,42 +157,40 @@ def test_get_all_auth_rule_transactions_after_write(looper,
             assert auth_map[key].as_dict == rule[CONSTRAINT]
 
 
-def test_auth_rule_after_get_auth_rule_without_changes(looper,
-                                       sdk_wallet_trustee,
-                                       sdk_pool_handle):
+def test_auth_rule_after_get_auth_rule_without_changes(
+    looper, sdk_pool_handle, sdk_wallet_trustee
+):
     # get all auth rules
-    auth_rules = sdk_get_auth_rule_request(looper,
-                                           sdk_wallet_trustee,
-                                           sdk_pool_handle)
-
-    # prepare action key
-    dict_key = dict(auth_rules[0][1][RESULT][DATA][0])
-    dict_key.pop(CONSTRAINT)
-
-    # prepare "operation" to send AUTH_RULE txn
-    op = dict(auth_rules[0][1][RESULT][DATA][0])
-    op[TXN_TYPE] = AUTH_RULE
-
-    # send AUTH_RULE txn
-    req_obj = sdk_gen_request(op, identifier=sdk_wallet_trustee[1])
-    req = sdk_sign_and_submit_req_obj(looper,
-                                      sdk_pool_handle,
+    auth_rules_resp = sdk_send_and_check_get_auth_rule_request(
+        looper, sdk_pool_handle, sdk_wallet_trustee
+    )
+    auth_rules = auth_rules_resp[0][1][RESULT][DATA]
+
+    for rule in auth_rules:
+        # prepare action key
+        dict_key = dict(rule)
+        dict_key.pop(CONSTRAINT)
+
+        # prepare AUTH_RULE request
+        req_json = build_auth_rule_request_json(
```

Comment:
The goal of the test is to make sure that we can send GET_AUTH_RULE response to AUTH_RULE `as-is`.
But, as I can see, `build_auth_rule_request_json` re-builds the AUTH_RULE json from the dict.
Maybe we can have two tests for GET - SET case: one we send the result as-is, and one when we use `build_auth_rule_request_json`.